it/uk/gov/hmrc/minorentityidentificationfrontend/stubs/StorageStub.scala

```diff
@@ -33,16 +33,35 @@ trait StorageStub extends WiremockMethods {
       status = status
     )
 
+  def stubStoreStoreTrustsKnownFacts(journeyId: String)(status: Int, body: JsValue = Json.obj()): Unit =
+    when(method = PUT, uri = s"/minor-entity-identification/journey/$journeyId/trustKnownFacts").thenReturn(
+      status = status,
+      body = body
+    )
+
+  def stubStoreIdentifiersMatch(journeyId: String)(status: Int, identifiersMatch: KnownFactsMatchingResult): Unit =
+    when(method = PUT, uri = s"/minor-entity-identification/journey/$journeyId/identifiersMatch").thenReturn(
+      status = status,
+      body = Json.obj("identifiersMatch" -> identifiersMatch)
```

Comment:
Same as above here

Reply:
... I did not get it Milly ... could you explain it please